### ISSUE TITLE
Time to run the gardener again

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/java-docs-samples#1224]
APIs are several versions out of date. E.g. 1.38.0 for translate

**Comment by kurtisvg** [comment]
Notice: this issue has been closed because it has been inactive for 250 days. You may reopen this issue if it has been closed in error.

**Comment by elharo** [comment]
No, I can't reopen this but it doesn't seem to have been resolved.
